scripts/busybox.sh

```diff
@@ -47,6 +47,13 @@ while true; do
 done
 
 :	${usrdir:="${PWD}"/usr}
+
```

Comment:
when i launch busybox.sh, he return: i have try with you're update $(usrdir:=${PWD}/usr}  

cd /usr/share/mkinitramfs-ll/scripts
./busybox.sh
./busybox.sh line 60: /usr/share/mkinitramfs-ll/scripts/usr/lib/mkinitramfs-ll/functions: no file or directory